modules/pricer/src/main/java/com/opengamma/strata/pricer/curve/PresentValueCalibrationMeasure.java

```diff
@@ -66,7 +69,17 @@
           (trade, p) -> DiscountingIborFutureTradePricer.DEFAULT.presentValueSensitivity(trade, p));
 
   /**
-   * The calibrator for {@link SwapTrade} using par spread discounting.
+   * The calibrator for {@link OvernightFutureTrade} using present value discounting.
+   */
+  public static final PresentValueCalibrationMeasure<ResolvedOvernightFutureTrade> OVERNIGHT_FUTURE_PV =
+      PresentValueCalibrationMeasure.of(
+          "OvernightFutureParSpreadDiscounting",
```

Comment:
These constants still all have "ParSpread" in the name - is that meaningful or need to be changed?

Reply:
Good spot